Sofascore2025/Sofascore2025/Model/Event.swift

```diff
@@ -0,0 +1,12 @@
+import Foundation
+
+struct Event: Codable {
+    let id: Int
+    let homeTeam: Team
+    let awayTeam: Team
+    let startTimestamp: Int
+    let status: String
```

Comment:
e ovo ti sad moze bit tip ovog
```suggestion
    let status: EventMatchStatus
```